Homework_1.py

```diff
@@ -2,42 +2,109 @@
 
 import pandas as pd
 import plotly.express as px
+from sklearn import datasets, metrics
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.model_selection import train_test_split
 from sklearn.preprocessing import StandardScaler
 
-data = pd.read_csv("/home/osboxes/Downloads/iris.csv")
-df1 = pd.DataFrame(data)
-df = pd.DataFrame(data)
-df.pop("class")
+# 1. loading the iris data using Pandas DataFrame
+iris = pd.DataFrame()
+x_scaled = []
+x = []
+y = []
 
-n_array = df.to_numpy()
 
-print(n_array)
+def load_database():
+    global iris
+    iris = datasets.load_iris()
```

Comment:
load_iris returns a dict object, whereas iris is a dataframe object.

Reply:
added iris as a global variable.

Reply:
added iris_data as an object and converted to dataframe called iris.